institutions-api/Jenkinsfile

```diff
@@ -26,7 +26,9 @@ volumes: [
               sh "docker tag ${env.DOCKER_HUB_USER}/institutions-api ${env.DOCKER_HUB_USER}/institutions-api:${gitBranch}"
               sh "docker login -u ${env.DOCKER_HUB_USER} -p ${env.DOCKER_HUB_PASSWORD} "
               sh "docker push ${env.DOCKER_HUB_USER}/institutions-api:${gitBranch}"
-              sh "docker rmi -f ${env.DOCKER_HUB_USER}/institutions-api:${gitBranch}"
+              sh "docker images"
+              sh "docker images | grep 'institutions-api' | awk '{print \$3}' | uniq"
+              sh "docker rmi -f \$(docker images | grep 'institutions-api' | awk '{print \$3}' | uniq)"
```

Comment:
How would this respond if there was a running `institutions-api` from kubernetes on the node where this jenkins pipeline is run?

Reply:
We're deleting the `image` and not the `container`. If an `institutions-api` container was running, it wouldn't get affected by deleting the `image`.